src/snowflake/snowpark/files.py

```diff
@@ -62,7 +62,7 @@ def close(self) -> None:
         """
         return
 
-    def detach(self) -> None:
+    def detach(self) -> None:  # pr
```

Comment:
nit: incomplete comment?

Reply:
good catch. I removed it. This method is actually covered.